game_patch/rf/multi.h

```diff
@@ -19,23 +19,30 @@ namespace rf
 
     // nw/psnet
 
+    #pragma pack(push, 4)
     struct NetAddr
     {
         uint32_t ip_addr;
         uint16_t port;
 
         bool operator==(const NetAddr &other) const = default;
     };
+    #pragma pack(pop)
     static_assert(sizeof(NetAddr) == 0x8);
 
-    static auto& net_init_socket = addr_as_ref<void(unsigned short port)>(0x00528F10);
-    static auto& net_addr_to_string = addr_as_ref<void(char *buf, int buf_size, const NetAddr& addr)>(0x00529FE0);
-    static auto& net_send = addr_as_ref<void(const NetAddr &addr, const void *data, int len)>(0x0052A080);
-    static auto& net_same = addr_as_ref<int(const NetAddr &addr1, const NetAddr &addr2, bool check_port)>(0x0052A930);
+    constexpr int NET_MAX_REL_SOCKETS = 40;
 
+    static const auto& net_init_socket = addr_as_ref<void(unsigned short port)>(0x00528F10);
+    static const auto& net_addr_to_string = addr_as_ref<void(char *buf, int buf_size, const NetAddr& addr)>(0x00529FE0);
```

Comment:
The NetAddr params for this and `net_send` and `net_same` are `NetAddr*` in code but `NetAddr&` here - why? I believe these should be written in pointer form.

Reply:
They cannot be null so we should encode this invariant.